test/sql/function_calls.sql

```diff
@@ -0,0 +1,615 @@
+
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            returnsAccountWithId(idToSearch: 1) {
+                email
+            }
+        }
+    $$));
+
+    create function returns_setof_account(top int)
+        returns setof account language sql stable
+    as $$ select id, email from account limit top; $$;
+
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            returnsSetofAccount(top: 2, last: 1) {
+                pageInfo {
+                    startCursor
+                    endCursor
+                    hasNextPage
+                    hasPreviousPage
+                }
+                edges {
+                    cursor
+                    node {
+                        nodeId
+                        id
+                        email
+                        __typename
+                    }
+                    __typename
+                }
+                totalCount
+                __typename
+            }
+        }
+    $$));
+
+    -- functions with args named `first`, `last`, `before`, `after`, `filter`, or `orderBy` are not exposed
+    create function arg_named_first(first int)
+        returns setof account language sql stable
+    as $$ select id, email from account; $$;
+
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            argNamedFirst {
+                __typename
+            }
+        }
+    $$));
+
+    create function arg_named_last(last int)
+        returns setof account language sql stable
+    as $$ select id, email from account; $$;
+
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            argNamedLast {
+                __typename
+            }
+        }
+    $$));
+
+    create function arg_named_before(before int)
+        returns setof account language sql stable
+    as $$ select id, email from account; $$;
+
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            argNamedBefore {
+                __typename
+            }
+        }
+    $$));
+
+    create function arg_named_after(after int)
+        returns setof account language sql stable
+    as $$ select id, email from account; $$;
+
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            argNamedAfter {
+                __typename
+            }
+        }
+    $$));
+
+    create function arg_named_filter(filter int)
+        returns setof account language sql stable
+    as $$ select id, email from account; $$;
+
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            argNamedFilter {
+                __typename
+            }
+        }
+    $$));
+
+    create function arg_named_orderBy(orderBy int)
+        returns setof account language sql stable
+    as $$ select id, email from account; $$;
+
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            argNamedOrderBy {
+                __typename
+            }
+        }
+    $$));
```

Comment:
since `arg_named_orderBy` and the arg `orderBy` contain a capital letter this is interpreted as `arg_named_orderby` with argument `orderby` and is being reflected in the GraphQL schema as `argNamedOrderby`

double quoting the function and arg name should resolve that

Reply:
Fixed.